app/gui/view/examples/interface/src/lib.rs

```diff
@@ -256,8 +256,10 @@ fn init(app: &Application) {
 
     // === Execution Modes ===
 
-    graph_editor
-        .set_available_execution_modes(vec!["development".to_string(), "production".to_string()]);
+    graph_editor.set_available_execution_environments(vec![
+        "Live".to_string().into(),
+        "Design".to_string().into(),
```

Comment:
yet another place with mode names. I know this is an example, but as we are keeping it sync'ed with the names, it should probably use the same names defined in one place.

Reply:
(see above)